src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -60,7 +60,7 @@ public void testDot()
         final String[] expected = {
             "31:10: " + getCheckMessage(MSG_LINE_NEW, "."),
         };
-        verify(checkConfig, getPath("InputSeparatorWrap.java"), expected);
+        verify(checkConfig, getPath("InputSeparatorWrapAfter3904.java"), expected);
```

Comment:
Do not make input file names cryptic. Please rename the file to clearly state its purpose.